web_client/src/store/index.ts

```diff
@@ -494,11 +494,6 @@ const {
       }
       if (state.currentProject && taskOverview.project_id === state.currentProject.id) {
         state.currentTaskOverview = taskOverview;
-        Object.values(store.state.scans).forEach((scan: Scan) => {
-          if (taskOverview.scan_states[scan.id] && taskOverview.scan_states[scan.id] !== 'unreviewed') {
-            store.dispatch.reloadScan(scan.id);
```

Comment:
So there was a chance that calling `reloadScan` could eventually cause this `setTaskOverview` to run?

Reply:
We know that refreshing the page will cause `loadProject` to execute, which calls `setTaskOverview`. I was thinking that the unreviewed state of these scans would cause `reloadScan` to execute, possibly causing an infinite loop. But it appears this is not the cause, as the problem is still there.

Reply:
This is the initiator for the `taskOverview` request that hangs: 
![image](https://user-images.githubusercontent.com/44912689/198371902-349827a8-288c-4b90-80e8-6f00dfd0d0c6.png)